source/UIAHandler/browseMode.py

```diff
@@ -493,6 +493,12 @@ def _iterNodesByType(self,nodeType,direction="next",pos=None):
 				UIAHandler.UIA_ControlTypePropertyId, UIAHandler.UIA_TabItemControlTypeId
 			)
 			return UIAControlQuicknavIterator(nodeType, self, pos, condition, direction)
+		elif nodeType == "progressBar":
+			condition = UIAHandler.handler.clientObject.createPropertyCondition(
+				UIAHandler.UIA_ControlTypePropertyId, UIAHandler.UIA_ProgressBarControlTypeId

```

Comment:
Please separate arguments on new lines for readability

```suggestion
				UIAHandler.UIA_ControlTypePropertyId,
				UIAHandler.UIA_ProgressBarControlTypeId
```